### ISSUE TITLE
have you tested your model with your own data?

### ISSUE THREAD

**Issue by BuaaAlban** [SeanNaren/deepspeech.pytorch#424]
I have trained the model with Librispeech clean dataset, LSTM with 678 hidden states, and i have got a WER 11 with libri  clean test dataset , 30 with noisy dataset, but it seems null for my own recording recorded in a quiet space, is the dataset not big enough or I have trained too much epoch(with loss 0) so that its overfit? All in all, the model is not that robust. What about yours?

**Comment by SeanNaren** [comment]
It does not perform well at all, as mentioned in the release README you'll need to train on much more data which is in domain to your production data. Check the DeepSpeech2 paper for more guidance on hours and sourcing of data!

**Comment by xieyidi** [comment]
Can I get your code as a reference, my training cannot converge
